{"uuid": "26f9d985-1edb-401b-85b3-f988ae914c93", "children": ["05d7a15b-5ec7-43fd-b17c-85a6e51a9cb9"], "befores": [{"name": "screenshot_on_failure", "status": "passed", "start": 1597851426409, "stop": 1597851426409}], "afters": [{"name": "screenshot_on_failure::fin", "status": "broken", "statusDetails": {"message": "AttributeError: 'Function' object has no attribute 'rep_setup'\n", "trace": "  File \"c:\\users\\m.kononova\\appdata\\local\\programs\\python\\python38-32\\lib\\site-packages\\allure_commons\\_allure.py\", line 185, in __call__\n    return self._fixture_function(*args, **kwargs)\n  File \"C:\\Users\\m.kononova\\environments\\flexite_autotests\\conftest.py\", line 44, in fin\n    if request.node.rep_setup.failed:\n"}, "start": 1597851433167, "stop": 1597851433394}], "start": 1597851426409, "stop": 1597851433394}